### ISSUE TITLE
Carbon Radio Button Selection not visible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by carmacleod** [carbon-design-system/carbon#1600]
1. Find a Windows machine, or use a Windows VM
2. Open either IE, Edge, or Firefox (Chrome does not show the problem because it does not honor Windows high contrast mode... instead it relies on 3rd party extensions to do high contrast).
3. Go to the [Carbon Radio Button page](https://www.carbondesignsystem.com/components/radio-button/code).
4. Type `left Alt + left Shift + PrintScreen` (then Yes) to [turn on Windows high contrast mode](https://support.microsoft.com/en-ca/help/13862/windows-use-high-contrast-mode)
5. Select one of the radio buttons.
6. You can't see the blue dot, so there's no way to know that the radio is selected.
(This is probably because background colors and background images are ignored in high contrast mode)

Note that the [example radio buttons in the ARIA Practices Guide](http://w3c.github.io/aria-practices/examples/radio/radio-1/radio-1.html) work correctly in Windows High Contrast mode. 

![image](https://user-images.githubusercontent.com/3331913/50850174-1c4e2580-1347-11e9-8b6d-64c7bd675187.png)

Notes:
- For more about why Windows High Contrast mode is important, [this slack conversation](https://ibm-cloudplatform.slack.com/archives/C036P1CTN/p1542898578381600) might be helpful.

**Comment by carmacleod** [comment]
Still a problem.
Note point 2 in  initial comment: IE, Edge, or Firefox. Not Chrome.

**Comment by dakahn** [comment]
Yep! We've addressed this upstream [in the elements repo](https://github.com/carbon-design-system/carbon-elements/pull/388) and are waiting on those fixes to get utilized in the core repos.

You can track that work here
https://github.com/carbon-design-system/carbon-components/issues/2104

**Comment by joshblack** [comment]
@dakahn is the radio button an icon in this case? Seems like it's implemented in CSS and will need a fix in there, as a result.
